src/app/icd/server/ICDConfigurationData.h

```diff
@@ -92,7 +96,22 @@ class ICDConfigurationData
 
 private:
     // Singleton Object
-    ICDConfigurationData() = default;
+    ICDConfigurationData()
+    {
+        // Initialize feature map
+#if CHIP_CONFIG_ENABLE_ICD_LIT
```

Comment:
Again, this is assuming that CHIP_CONFIG_ENABLE_ICD_CIP is only allowed when CHIP_CONFIG_ENABLE_ICD_LIT, which is just not true.

Reply:
Moved the CIP and UAT out of the LITS